reflex/compiler/utils.py

```diff
@@ -290,7 +290,7 @@ def create_theme(style: ComponentStyle) -> dict:
         The base style for the app.
     """
     # Get the global style from the style dict.
-    style_rules = Style({k: v for k, v in style.items() if not isinstance(k, type)})
+    style_rules = Style({k: v for k, v in style.items() if not isinstance(k, Callable)})
```

Comment:
```suggestion
    style_rules = Style({k: v for k, v in style.items() if not isinstance(k, (type, Callable))})
```

lets support both, so we don't break existing apps

we can even have a thing like

```python
for style_key in style_rules:
    if isinstance(style_key, type):
        console.deprecate(
            feature_name="Component classes in global styles",
            reason="Use the lowercase component function, like rx.text",
            deprecation_version="0.3.10",
            removal_version="1.0",
        )
```

Reply:
Agreed that we don't want to break any existing code

Reply:
ah okay, sounds good 👍